sources/osgShader/Compiler.js

```diff
@@ -775,6 +783,34 @@ define( [
 
         },
 
+        createShadowTextureInputVarying: function ( shadowTexture, inputs, vertexWorld, tUnit ) {
+            var shadowTexSamplerName = 'Texture' + tUnit;
+
+            // per texture uniforms
+            var shadowTextureUniforms = this.getOrCreateTextureStateAttributeUniforms( shadowTexture, 'shadowTexture', tUnit );
+
+            // UGLY REMOVAL
+            // TODO: create a specific texture getuniform for compiler
+            var id = shadowTextureUniforms[ 'shadowTexture' + shadowTexSamplerName ].getID();
```

Comment:
no need, didn't change other uniform name, juste texture name ( it's the shadowTexSamplerName 2 line above I had to change)